server/storage/kv/kv_test.go

```diff
@@ -23,83 +23,79 @@ import (
 	"strconv"
 	"testing"
 
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/require"
 	"github.com/tikv/pd/pkg/tempurl"
 	"go.etcd.io/etcd/clientv3"
 	"go.etcd.io/etcd/embed"
 )
 
-func TestKV(t *testing.T) {
-	TestingT(t)
-}
-
-type testKVSuite struct{}
-
-var _ = Suite(&testKVSuite{})
-
-func (s *testKVSuite) TestEtcd(c *C) {
+func TestEtcd(t *testing.T) {
+	re := require.New(t)
 	cfg := newTestSingleConfig()
 	defer cleanConfig(cfg)
 	etcd, err := embed.StartEtcd(cfg)
-	c.Assert(err, IsNil)
+	re.NoError(err)
 	defer etcd.Close()
 
 	ep := cfg.LCUrls[0].String()
 	client, err := clientv3.New(clientv3.Config{
 		Endpoints: []string{ep},
 	})
-	c.Assert(err, IsNil)
+	re.NoError(err)
 	rootPath := path.Join("/pd", strconv.FormatUint(100, 10))
 
 	kv := NewEtcdKVBase(client, rootPath)
-	s.testReadWrite(c, kv)
-	s.testRange(c, kv)
+	testReadWrite(t, kv)
+	testRange(t, kv)
 }
 
-func (s *testKVSuite) TestLevelDB(c *C) {
+func TestLevelDB(t *testing.T) {
+	re := require.New(t)
 	dir, err := os.MkdirTemp("/tmp", "leveldb_kv")
-	c.Assert(err, IsNil)
+	re.NoError(err)
 	defer os.RemoveAll(dir)
 	kv, err := NewLevelDBKV(dir)
-	c.Assert(err, IsNil)
+	re.NoError(err)
 
-	s.testReadWrite(c, kv)
-	s.testRange(c, kv)
+	testReadWrite(t, kv)
+	testRange(t, kv)
 }
 
-func (s *testKVSuite) TestMemKV(c *C) {
+func TestMemKV(t *testing.T) {
 	kv := NewMemoryKV()
-	s.testReadWrite(c, kv)
-	s.testRange(c, kv)
+	testReadWrite(t, kv)
+	testRange(t, kv)
 }
 
-func (s *testKVSuite) testReadWrite(c *C, kv Base) {
+func testReadWrite(t *testing.T, kv Base) {
+	re := require.New(t)
```

Comment:
What about just passing `re *require.Assertions` here to avoid unnecessary `require.New(t)`?

Reply:
Good idea~